src/tests/Microsoft.PowerFx.Core.Tests.Shared/IntellisenseTests/SignatureHelpTest.cs

```diff
@@ -24,13 +24,48 @@ public class SignatureHelpTest : IntellisenseTestBase
         /// instead include the subpath bin/(Debug|Release).AnyCPU, depending on whether the assembly was
         /// built in debug or release mode.
         /// </summary>
+
+#if NET8_0
+        private static readonly string _baseDirectory = Path.Join(Directory.GetCurrentDirectory(), "IntellisenseTests", "TestSignatures");
+
+        private static readonly string _signatureHelpDirectory = RegenerateSignatureHelp
+            ? _baseDirectory
+                .Replace(Path.Join("bin", "Debug", "net8.0"), string.Empty)
+                .Replace(Path.Join("bin", "Release", "net8.0"), string.Empty)
+                .Replace(Path.Join("bin", "DebugAll", "net8.0"), string.Empty)
+                .Replace(Path.Join("bin", "ReleaseAll", "net8.0"), string.Empty)
+                .Replace(Path.Join("bin", "Debug80", "net8.0"), string.Empty)
+                .Replace(Path.Join("bin", "Release80", "net8.0"), string.Empty)
+            : _baseDirectory;
```

Comment:
Can we move this to shim / some more centralized place where it's more obvious to update